inc/ProgEvent.h

```diff
@@ -126,6 +124,10 @@ namespace PCOE {
          *  @see        ProgEvents
          **/
         void setNPoints(const unsigned int nPoints);
+
+    private:
+        unsigned int nSamples; ///< Number of samples in the occurrence matrix
+        ///@todo(CT): IS THIS NECESSARY?
```

Comment:
Old, but is what necessary? Can we resolve this or make it more descriptive?

Reply:
We can remove this comment- no longer relevant